tests/test_migrator.py

```diff
@@ -1,9 +1,35 @@
-# -*- coding: utf-8 -*-
+# from mock import patch
 
-import unittest
+from migrate_metedata.migrator import Migrator
+import metastore.backend as store
 
-import ckan_datapackage_tools.converter as converter
+migrator_obj = Migrator('https://demo.ckan.org/api/3/action', 'filesystem', dict(uri='mem://'))
```

Comment:
You should not be sharing your tested object between tests. In each test, create a fresh migrator object to test, this way you ensure you are not "polluting" tests with pre-existing state. If you need pre-existing state (e.g. to test what happens with conflicting package names in metastore), you should either create it as part of the test or have some kind of fixture (read a bit about Pytest fixtures) creating a pre-existing storage state and destroying it after each test.